src/Hosting/Hosting/src/Internal/StartupLoader.cs

```diff
@@ -228,15 +228,17 @@ public static Type FindStartupType(string startupAssemblyName, string environmen
             if (string.IsNullOrEmpty(startupAssemblyName))
             {
                 throw new ArgumentException(
-                    string.Format("A startup method, startup type or startup assembly is required. If specifying an assembly, '{0}' cannot be null or empty.",
-                    nameof(startupAssemblyName)),
-                    nameof(startupAssemblyName));
+                    string.Format(
+                        CultureInfo.CurrentCulture,
+                        "A startup method, startup type or startup assembly is required. If specifying an assembly, '{0}' cannot be null or empty.",
+                        nameof(startupAssemblyName)),
+                        nameof(startupAssemblyName));
             }
 
             var assembly = Assembly.Load(new AssemblyName(startupAssemblyName));
             if (assembly == null)
             {
-                throw new InvalidOperationException(String.Format("The assembly '{0}' failed to load.", startupAssemblyName));
+                throw new InvalidOperationException($"The assembly '{startupAssemblyName}' failed to load.");
```

Comment:
I think it's fine, but why change it from string.Format?

Reply:
Is this another code analysis case?

Reply:
I have a slight preference to using string interpolation in places that allow it. It is equivalent to the latter with `CultureInfo.CurrentCulture`.